dev_environment/l3af_on_windows.md

```diff
@@ -0,0 +1,81 @@
+### Prerequisites:
+
+- [Git](https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.3/Git-2.41.0.3-64-bit.exe)
+- Cmake
+- Clang
+- nmake
+- [Golang](https://aka.ms/vs/17/release/vs_buildtools.exe)
+- C++ using: [MSYS2](https://www.msys2.org/)
```

Comment:
Would be good to know why MSYS2 is needed, so we can work to remove that dependency